src/Tab.js

```diff
@@ -109,13 +119,23 @@ const Tab = ({ icon, onClick, selected, disabled, children }) => (
     </button>
 )
 
+/**
+ * @typedef {Object} PropTypes
+ * @static
+ * @prop {Element} [icon]
+ * @prop {function} [onClick]
+ * @prop {boolean} [selected]
+ * @prop {boolean} [disabled]
+ * @prop {Node} [children]
+ * @prop {string} [className]
+ */
 Tab.propTypes = {
-    icon: PropTypes.element,
-    onClick: PropTypes.func,
-    addTabRef: PropTypes.func,
```

Comment:
Possible breaking change, addTabRef was dropped.

Reply:
It was unused so ESLint complained.